sass/_chatbox.scss

```diff
@@ -121,6 +121,20 @@
             width: var(--mobile-chat-width);
         }
 
+        #service_administration_description {
+            color: #777;
+            background-color:white;
+            display:block;
+            padding:5px;
```

Comment:
Use `em` for padding. That way padding is relative to the font-size and scales when people make their browser text larger or smaller (e.g with <ctrl> - and <ctrl> =).